ui/raidboss/data/06-ew/raid/p1n.ts

```diff
@@ -89,8 +89,8 @@ const triggerSet: TriggerSet<Data> = {
       type: 'GainsEffect',
       netRegex: NetRegexes.gainsEffect({ effectId: ['AB3', 'AB4'], capture: true }),
       condition: Conditions.targetIsYou(),
-      alertText: (_data, _matches, _output) => {
-        return _matches.effectId === 'AB3' ? _output.red!() : _output.blue!();
+      alertText: (_data, _matches, output) => {
+        return _matches.effectId === 'AB3' ? output.red!() : output.blue!();
```

Comment:
Renamed parameters here and below to not be underscore-prefixed, so that they would properly get picked up by test_trigger.ts.